### PR TITLE
update script to add restore plugin cmd

### PR DESCRIPTION
**Pull request by hongzhen-ma**
#### What type of this PR
- Bug fixes

####
1、update env para in plugin cmd
2、sync plugin cmd

#### Which issue(s) this PR fixes:
Fixes #(issue-number)

### CODE REVIEW

File: dist/images/update/1.7-1.8.2.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
```
Comment (oilbeater) [inline dist/images/update/1.7-1.8.2.sh:1]: the file should renamed to 1.7-1.8.3.sh

File: dist/images/update/1.8.0-1.8.2.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
```
Comment (oilbeater) [inline dist/images/update/1.8.0-1.8.2.sh:1]: file rename

File: dist/images/update/1.8.2-1.9.0.sh [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
```
Comment (oilbeater) [inline dist/images/update/1.8.2-1.9.0.sh:1]: file rename

**Review by oilbeater — CHANGES_REQUESTED** [review]

File: dist/images/update/1.7-1.8.2.sh (hunk 1 above)
Comment (hongzhen-ma) [reply dist/images/update/1.7-1.8.2.sh:1]: modifief

File: dist/images/update/1.8.2-1.9.0.sh (hunk 3 above)
Comment (hongzhen-ma) [reply dist/images/update/1.8.2-1.9.0.sh:1]: modified

**Review by oilbeater — APPROVED** [review]

File: dist/images/update/1.8.0-1.8.2.sh (hunk 2 above)
Comment (hongzhen-ma) [reply dist/images/update/1.8.0-1.8.2.sh:1]: modified

**Pull request merged**